### PR TITLE
qemu.tests: Small fix in invalid_parameter.py

### PR DESCRIPTION
**Pull request by spcui**
Have no attribute 'reason' in class VMCreateError().

Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1218500

### CODE REVIEW

**Comment by spcui** [comment]
PASS smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.invalid_parameter.fd_vhostfds

File: qemu/tests/invalid_parameter.py [hunk 1]
```diff
@@ -25,10 +25,10 @@ def run(test, params, env):
         vm.destroy()
     except Exception, emsg:
         error.context("Check guest exit status.")
-        if "(core dumped)" in emsg.reason:
+        if "(core dumped)" in emsg.output:
```
Comment (xutian) [inline qemu/tests/invalid_parameter.py]: Why not 'core dmped' in str(emsg) ?

**Comment by spcui** [comment]
@xutian updated with your comment, and passed by testing.

**Comment by PandaWei** [comment]
Indeed, Using msg looks better.
ACK

**Comment by xutian** [comment]
Acked,

Thanks @spcui  and @PandaWei

**Pull request merged**